.github/workflows/release-go-master.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and Release master branch
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest] #, macos-latest, windows-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-go@v1
+        with:
+          go-version: '1.14.x'
+      - run: |
+          export GIT_COMMIT=$(git rev-parse HEAD) && \
+          export GIT_REPOSITORY=$(git config --get remote.origin.url) && \
+          CGO_ENABLED=0 go build -ldflags "-X github.com/SAP/jenkins-library/cmd.GitCommit=${GIT_COMMIT} \
+                  -X github.com/SAP/jenkins-library/pkg/log.LibraryRepository=${GIT_REPOSITORY} \
+                  -X github.com/SAP/jenkins-library/pkg/telemetry.LibraryRepository=${GIT_REPOSITORY}" \
+            -o piper_${{ matrix.os }}_master .
+        shell: bash
+      - run: ./piper-${{ matrix.os }} version
+        shell: bash
+      - uses: actions/upload-artifact@v1
+        with:
+          name: piper-${{ matrix.os }}
+          path: piper-${{ matrix.os }}
+        #currently works only on linux because only linux binaries are available to the action
+      - uses: SAP/project-piper-action@master
+        with:
+         command: githubPublishRelease
+         flags: -v --token ${{ secrets.GITHUB_TOKEN }} --owner fwilhe --version latest --assetPath ./piper_${{ matrix.os }}_master
```

Comment:
need to update `owner`

Reply:
this changes the artifact name in the release which turns it unusable in the current download steps.
We should stick with `piper` for now ...

Reply:
Yes, I've left the owner to my fork because I am not sure if this would then do a release on upstream and I want to avoid that.

Also I agree with the name, I've started with a matrix build  on mac windows and linux, but I see we're currently not interested in shipping non-linux binaries, so that's kind of a leftover. Still I think we should do that at some point.

Reply:
sorry, overlooked the comment above ...